index.html

```diff
@@ -4666,7 +4773,7 @@ <h2 id="AudioBufferSourceNode">
             </p>
             <dl class="parameters">
               <dt>
-                optional double when = 0
+                double when
```

Comment:
I don't get why we are leaving `start()`in this interface if it's inherited from `AudioScheduledSourceNode`. And why would `when` no longer be optional for an ABSN's implementation of this method?